### PR TITLE
Adds two new evac shelter basements and vandalize them

### PR DESCRIPTION
**Pull request by haveric**
#### Summary
Content "Adds two new evac shelter basements and vandalize them"

#### Purpose of change

- Add two new evac shelter basements
- Make basements vandalized as well as the main floor
- Made zombies spawning in vandalized basements consistently dangerous
- Cleanup/Organize the shelter json

#### Describe the solution

- Utilize palettes better to keep shelter spawning consistent.
- Changed zombie spawns to force intensity, which guarantees they spawn when they should. (It's only 10% of used/vandalized evac shelters and only in the basements)

#### Describe alternatives you've considered

None

#### Testing

Spawned in several evac shelters of each variant and in each damage state. Verified they looked good. Teleported around to a few shelters to make sure they are generating correctly in the wild, which they are. Also verified the evac computer still operates since I moved that to a nested mapgen.

Also tested that No Hope and Dark Skies Above shelters are spawning correctly. I did my best to keep these as intended by the mod authors, but will update anything for either as necessary.

#### Additional context

Basement Screenshots (There are two copies of each one vertically):
Default:
<details><summary>Evac 1 - Evac 2 (Inglonias)</summary>

image
</details>
<details><summary>Evac 3 - Evac 4 (Both mine)</summary>

image
</details>
Used:
<details><summary>Evac 1 - Evac 2 (Inglonias)</summary>

image
</details>
<details><summary>Evac 3 - Evac 4 (Both mine)</summary>

image
</details>
Vandalized:
<details><summary>Evac 1 - Evac 2 (Inglonias)</summary>

image
</details>
<details><summary>Evac 3 - Evac 4 (Both mine)</summary>

image
</details>

Main floor (Making sure I didn't break anything)
<details><summary>Default:</summary>

image
image
</details>
<details><summary>Used:</summary>

image
image
</details>
<details><summary>Vandalized:</summary>

image
image
</details>

### CODE REVIEW

File: data/json/mapgen_palettes/shelter.json [hunk 1]
```diff
@@ -27,13 +28,156 @@
...
+      ";": { "chunks": [ [ "shelter_graffiti", 5 ], [ "general_graffiti", 20 ], [ "null", 75 ] ] },
+      "/": { "chunks": [ [ "shelter_graffiti", 5 ], [ "general_graffiti", 20 ], [ "null", 75 ] ] },
+      "6": { "chunks": [ [ "shelter_computer", 50 ], [ "shelter_computer_broken", 50 ] ] },
+      "x": { "chunks": [ [ "shelter_computer", 20 ], [ "shelter_computer_broken", 80 ] ] }
+    }
+  },
+  {
+    "type": "palette",
+    "id": "shelter_zombies",
+    "monsters": { "_": { "monster": "GROUP_ZOMBIE", "density": 0.0005 }, "b": { "monster": "GROUP_ZOMBIE", "density": 0.0001 } }
```
Comment (Inglonias) [inline data/json/mapgen_palettes/shelter.json:181]: I would be very careful with the density parameter here and verify what it does. If it does the same thing as it does in `place_monster` and `place_monsters`, then this flag will multiply with the world spawn rate and round to the nearest integer.

Comment (Inglonias) [reply data/json/mapgen_palettes/shelter.json:181]: Frankly, I would advise against using this JSON structure to place monsters, as it's difficult to control. Suggest you use the `place_monster` flag instead, as it lets you determine exactly how many times to roll the dice in a clear way.

Comment (haveric) [reply data/json/mapgen_palettes/shelter.json:181]: It seems to be placing a handful of zombies, which is what I intended. I'm trying to avoid having this be a completely crowded location, since it basically never spawned anything before. My assumption is that only a few people made it to these shelters and they then turned after the cataclysm.

Comment (Inglonias) [reply data/json/mapgen_palettes/shelter.json:181]: If you've run tests and things are working as intended, that's all well and good then. I just didn't want a situation like we had with hospitals and parks, where there was an absurd number of zombies.

Comment (haveric) [reply data/json/mapgen_palettes/shelter.json:181]: It's working well enough for me at least. The screenshots only show a couple zombies in them. I've only ever seen about 5 zombies spawn from it.

**Comment by Inglonias** [comment]
running tests, I'm getting an error message on starting a new game:

 DEBUG    : shelter_under_1 (struct oter_type_t) has two definitions from the same source (dda)!

 FUNCTION : check_duplicate_entries
 FILE     : D:\a\Cataclysm-DDA\Cataclysm-DDA\src\mod_tracker.h
 LINE     : 76
 VERSION  : 2eef1a1

**Comment by haveric** [comment]
@Inglonias I'm already working on it. It takes a while to compile sometimes.

**Comment by haveric** [comment]
We put our definitions in different orders, so the merge just accepted them at first. I just had to move my changes on top of yours to preserve your history. Also here's a 10x zombie spawn screenshot:
image

**Comment by Inglonias** [comment]
Again, monster spawning is odd to me. I spawned into a vandalized shelter in a world with a spawn rate of 1 and went to the basement. Then I used the debug menu's "spawn nested overmap" function for shelter_under_2_nest_zombies until it spawned something. It took eight to ten tries for it to spawn a single feral human. All of this is to say that I think those super-low densities are to blame for this behavior. Looking through the code shows that this is how spawn numbers are decided:```

if( MonsterGroupManager::is_animal( group ) ) {
        spawn_density = get_option< float >( "SPAWN_ANIMAL_DENSITY" );
    } else {
        spawn_density = get_option< float >( "SPAWN_DENSITY" );
    }

    float multiplier = density * spawn_density;
    // Only spawn 1 creature if individual flag set, else scale by density
    float thenum = individual ? 1 : ( multiplier * rng_float( 10.0f, 50.0f ) );
    int num = roll_remainder( thenum );
```

This suggests to me that densities of 0.0005 and 0.0001 are far, far, far too low for a good chance of getting monsters. Now, if that's what you were going for, that's fine. I'll stop bothering you about it if that's what you're going for, but considering that there's only a 10% chance of getting those dice to roll in the first place, getting a monster in the basement is supremely unlikely.

If you don't mind not having precise placement control of your monsters, I suggest using the `place_monster` flag instead of `monsters` in the palette. If you need that precise control, I suggest a new palette entry that pops up in only a few tiles and has a density of 1, maybe throwing a chance onto those tiles as well.

**Comment by haveric** [comment]
The previous behavior relied on the world's intensity, which from what I understand relates to how close you are to a city. This being a building that doesn't spawn in cities makes that never spawn any zombies, even though it was supposed to. I am happy with the low numbers of zombies spawning because I don't want these to become overwhelmed or full of zombies. This should be a light threat and I am happy with the behavior it is showing right now.

**Comment by Inglonias** [comment]
In that case, I'll drop my objections. I was trying to avoid a situation where not understanding the nonsense that these numbers go through before they spawn anything resulted in undesired behavior (see parks and hospitals in 0.F for an example)

**Comment by haveric** [comment]
> In that case, I'll drop my objections. I was trying to avoid a situation where not understanding the nonsense that these numbers go through before they spawn anything resulted in undesired behavior (see parks and hospitals in 0.F for an example)

I'm not going to pretend I fully understand them, but I referenced other locations' values and adjusted it a few times because there were too many with higher intensities and have tested it enough times that I'm fairly confident it's working as intended. Even with 10x scaling, it isn't too excessive. It can always be balanced in the future if there's some edge case that I'm not taking account of though.

**Comment by Inglonias** [comment]
any idea what's going on with the automated tests? It looks like they're being cancelled after six hours. I guess they're hanging?

**Comment by BrettDong** [comment]
This is the error message in the failed MODs loading test:
```
 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:634:15: format: rows: row 17 column 6: 'B' has no terrain, furniture, or other definition

        "####;S_%|b________b;####",
        "####;|+||b________b;####",
        "####;B
              ^
               ____________b;####",
        "####;|+||b________b;####",
        "####;S_%|b________b;####",

21:49:53.269 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:480:23: format: rows: row 9 column 14: 'U' has no terrain, furniture, or other definition

        "########################",
        "#######;;;;;;;;;;#######",
        "#######;lll||U
                      ^
                       UU;#######",
        "####;;;;|+||||+|;#######",
        "####;b__________;;;;####",

21:49:53.269 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:480:24: format: rows: row 9 column 15: 'U' has no terrain, furniture, or other definition

        "########################",
        "#######;;;;;;;;;;#######",
        "#######;lll||UU
                       ^
                        U;#######",
        "####;;;;|+||||+|;#######",
        "####;b__________;;;;####",

21:49:53.269 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:480:25: format: rows: row 9 column 16: 'U' has no terrain, furniture, or other definition

        "########################",
        "#######;;;;;;;;;;#######",
        "#######;lll||UUU
                        ^
                         ;#######",
        "####;;;;|+||||+|;#######",
        "####;b__________;;;;####",

21:49:53.270 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:483:26: format: rows: row 12 column 17: 't' has no terrain, furniture, or other definition

        "####;;;;|+||||+|;#######",
        "####;b__________;;;;####",
        "####;b_bb_bb____t
                         ^
                          tt;####",
        "####;b_bb_bb____bbb;####",
        "####;b_bb_bb_______;####",

21:49:53.270 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:483:27: format: rows: row 12 column 18: 't' has no terrain, furniture, or other definition

        "####;;;;|+||||+|;#######",
        "####;b__________;;;;####",
        "####;b_bb_bb____tt
                          ^
                           t;####",
        "####;b_bb_bb____bbb;####",
        "####;b_bb_bb_______;####",

21:49:53.270 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:483:28: format: rows: row 12 column 19: 't' has no terrain, furniture, or other definition

        "####;;;;|+||||+|;#######",
        "####;b__________;;;;####",
        "####;b_bb_bb____ttt
                           ^
                            ;####",
        "####;b_bb_bb____bbb;####",
        "####;b_bb_bb_______;####",

21:49:53.271 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:487:26: format: rows: row 16 column 17: 't' has no terrain, furniture, or other definition

        "####;b_bb_bb_______;####",
        "####;b_bb_bb____bbb;####",
        "####;b_bb_bb____t
                         ^
                          tt;####",
        "####;b_bb_bb____bbb;####",
        "####;||||||||______;####",

21:49:53.271 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:487:27: format: rows: row 16 column 18: 't' has no terrain, furniture, or other definition

        "####;b_bb_bb_______;####",
        "####;b_bb_bb____bbb;####",
        "####;b_bb_bb____tt
                          ^
                           t;####",
        "####;b_bb_bb____bbb;####",
        "####;||||||||______;####",

21:49:53.271 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:487:28: format: rows: row 16 column 19: 't' has no terrain, furniture, or other definition

        "####;b_bb_bb_______;####",
        "####;b_bb_bb____bbb;####",
        "####;b_bb_bb____ttt
                           ^
                            ;####",
        "####;b_bb_bb____bbb;####",
        "####;||||||||______;####",

21:49:53.272 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:490:15: format: rows: row 19 column 6: 'B' has no terrain, furniture, or other definition

        "####;b_bb_bb____bbb;####",
        "####;||||||||______;####",
        "####;B
              ^
               ______+___bbb;####",
        "####;|+|||+||___ttt;####",
        "####;T_%|%_T|___|||;####",

21:49:53.272 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:491:26: format: rows: row 20 column 17: 't' has no terrain, furniture, or other definition

        "####;||||||||______;####",
        "####;B______+___bbb;####",
        "####;|+|||+||___t
                         ^
                          tt;####",
        "####;T_%|%_T|___|||;####",
        "####;|_S|S_||___<///####",

21:49:53.272 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:491:27: format: rows: row 20 column 18: 't' has no terrain, furniture, or other definition

        "####;||||||||______;####",
        "####;B______+___bbb;####",
        "####;|+|||+||___tt
                          ^
                           t;####",
        "####;T_%|%_T|___|||;####",
        "####;|_S|S_||___<///####",

21:49:53.272 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:491:28: format: rows: row 20 column 19: 't' has no terrain, furniture, or other definition

        "####;||||||||______;####",
        "####;B______+___bbb;####",
        "####;|+|||+||___ttt
                           ^
                            ;####",
        "####;T_%|%_T|___|||;####",
        "####;|_S|S_||___<///####",

21:49:53.273 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:493:27: format: rows: row 22 column 18: '/' has no terrain, furniture, or other definition

        "####;|+|||+||___ttt;####",
        "####;T_%|%_T|___|||;####",
        "####;|_S|S_||___</
                          ^
                           //####",
        "####;T_S|S_T|___<///####",
        "####;;;;;;;;;;;;////####"

21:49:53.273 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:493:28: format: rows: row 22 column 19: '/' has no terrain, furniture, or other definition

        "####;|+|||+||___ttt;####",
        "####;T_%|%_T|___|||;####",
        "####;|_S|S_||___<//
                           ^
                            /####",
        "####;T_S|S_T|___<///####",
        "####;;;;;;;;;;;;////####"

21:49:53.273 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:493:29: format: rows: row 22 column 20: '/' has no terrain, furniture, or other definition

        "####;|+|||+||___ttt;####",
        "####;T_%|%_T|___|||;####",
        "####;|_S|S_||___<///
                            ^
                             ####",
        "####;T_S|S_T|___<///####",
        "####;;;;;;;;;;;;////####"

21:49:53.274 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:494:27: format: rows: row 23 column 18: '/' has no terrain, furniture, or other definition

        "####;T_%|%_T|___|||;####",
        "####;|_S|S_||___<///####",
        "####;T_S|S_T|___</
                          ^
                           //####",
        "####;;;;;;;;;;;;////####"
      ],

21:49:53.274 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:494:28: format: rows: row 23 column 19: '/' has no terrain, furniture, or other definition

        "####;T_%|%_T|___|||;####",
        "####;|_S|S_||___<///####",
        "####;T_S|S_T|___<//
                           ^
                            /####",
        "####;;;;;;;;;;;;////####"
      ],

21:49:53.274 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:494:29: format: rows: row 23 column 20: '/' has no terrain, furniture, or other definition

        "####;T_%|%_T|___|||;####",
        "####;|_S|S_||___<///####",
        "####;T_S|S_T|___<///
                            ^
                             ####",
        "####;;;;;;;;;;;;////####"
      ],

21:49:53.274 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:495:26: format: rows: row 24 column 17: '/' has no terrain, furniture, or other definition

        "####;|_S|S_||___<///####",
        "####;T_S|S_T|___<///####",
        "####;;;;;;;;;;;;/
                         ^
                          ///####"
      ],
      "palettes": [ "shelter" ]

21:49:53.275 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:495:27: format: rows: row 24 column 18: '/' has no terrain, furniture, or other definition

        "####;|_S|S_||___<///####",
        "####;T_S|S_T|___<///####",
        "####;;;;;;;;;;;;//
                          ^
                           //####"
      ],
      "palettes": [ "shelter" ]

21:49:53.275 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:495:28: format: rows: row 24 column 19: '/' has no terrain, furniture, or other definition

        "####;|_S|S_||___<///####",
        "####;T_S|S_T|___<///####",
        "####;;;;;;;;;;;;///
                           ^
                            /####"
      ],
      "palettes": [ "shelter" ]

21:49:53.275 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:495:29: format: rows: row 24 column 20: '/' has no terrain, furniture, or other definition

        "####;|_S|S_||___<///####",
        "####;T_S|S_T|___<///####",
        "####;;;;;;;;;;;;////
                            ^
                             ####"
      ],
      "palettes": [ "shelter" ]

21:49:53.277 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:551:21: format: rows: row 7 column 12: '/' has no terrain, furniture, or other definition

        "########################",
        "########################",
        "###########/
                    ^
                     ;;;;;;;;;###",
        "###;;;;;;;;;|T|T|T|T;###",
        "###;bbbbb|UU|_______;###",

21:49:53.277 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:553:20: format: rows: row 9 column 11: 'U' has no terrain, furniture, or other definition

        "###########/;;;;;;;;;###",
        "###;;;;;;;;;|T|T|T|T;###",
        "###;bbbbb|U
                   ^
                    U|_______;###",
        "###;_____||+|BCSCS_%;###",
        "##;;_bb_____||||||+|;;##",

21:49:53.277 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:553:21: format: rows: row 9 column 12: 'U' has no terrain, furniture, or other definition

        "###########/;;;;;;;;;###",
        "###;;;;;;;;;|T|T|T|T;###",
        "###;bbbbb|UU
                    ^
                     |_______;###",
        "###;_____||+|BCSCS_%;###",
        "##;;_bb_____||||||+|;;##",

21:49:53.278 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:554:23: format: rows: row 10 column 14: 'B' has no terrain, furniture, or other definition

        "###;;;;;;;;;|T|T|T|T;###",
        "###;bbbbb|UU|_______;###",
        "###;_____||+|B
                      ^
                       CSCS_%;###",
        "##;;_bb_____||||||+|;;##",
        "##;b_bb______________;##",

21:49:53.278 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:554:24: format: rows: row 10 column 15: 'C' has no terrain, furniture, or other definition

        "###;;;;;;;;;|T|T|T|T;###",
        "###;bbbbb|UU|_______;###",
        "###;_____||+|BC
                       ^
                        SCS_%;###",
        "##;;_bb_____||||||+|;;##",
        "##;b_bb______________;##",

21:49:53.278 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:554:26: format: rows: row 10 column 17: 'C' has no terrain, furniture, or other definition

        "###;;;;;;;;;|T|T|T|T;###",
        "###;bbbbb|UU|_______;###",
        "###;_____||+|BCSC
                         ^
                          S_%;###",
        "##;;_bb_____||||||+|;;##",
        "##;b_bb______________;##",

21:49:53.279 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:558:28: format: rows: row 14 column 19: 't' has no terrain, furniture, or other definition

        "##;b_bb______________;##",
        "##;b_bb_bb________bbb;##",
        "##;b_bb_b||||||___t
                           ^
                            tt;##",
        "##;b_bb_b|____|___bbb;##",
        "##;b_bb__+_<<_+___bbb;##",

21:49:53.279 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:558:29: format: rows: row 14 column 20: 't' has no terrain, furniture, or other definition

        "##;b_bb______________;##",
        "##;b_bb_bb________bbb;##",
        "##;b_bb_b||||||___tt
                            ^
                             t;##",
        "##;b_bb_b|____|___bbb;##",
        "##;b_bb__+_<<_+___bbb;##",

21:49:53.279 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:558:30: format: rows: row 14 column 21: 't' has no terrain, furniture, or other definition

        "##;b_bb______________;##",
        "##;b_bb_bb________bbb;##",
        "##;b_bb_b||||||___ttt
                             ^
                              ;##",
        "##;b_bb_b|____|___bbb;##",
        "##;b_bb__+_<<_+___bbb;##",

21:49:53.280 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:561:28: format: rows: row 17 column 19: 't' has no terrain, furniture, or other definition

        "##;b_bb_b|____|___bbb;##",
        "##;b_bb__+_<<_+___bbb;##",
        "##;b_bb__+_<<_+___t
                           ^
                            tt;##",
        "##;b_bb_b|____|___bbb;##",
        "##;b_bb_b||||||______;##",

21:49:53.280 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:561:29: format: rows: row 17 column 20: 't' has no terrain, furniture, or other definition

        "##;b_bb_b|____|___bbb;##",
        "##;b_bb__+_<<_+___bbb;##",
        "##;b_bb__+_<<_+___tt
                            ^
                             t;##",
        "##;b_bb_b|____|___bbb;##",
        "##;b_bb_b||||||______;##",

21:49:53.281 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:561:30: format: rows: row 17 column 21: 't' has no terrain, furniture, or other definition

        "##;b_bb_b|____|___bbb;##",
        "##;b_bb__+_<<_+___bbb;##",
        "##;b_bb__+_<<_+___ttt
                             ^
                              ;##",
        "##;b_bb_b|____|___bbb;##",
        "##;b_bb_b||||||______;##",

21:49:53.281 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:565:23: format: rows: row 21 column 14: 'B' has no terrain, furniture, or other definition

        "##;b_bb_b||||||______;##",
        "##;b_bb_bb|l||||||+||;##",
        "##;;______+l|B
                      ^
                       CSCS_%|/##",
        "###;bbbbbb|l|_______;/##",
        "###;;;;;;;;;;T|T|T|T;###",

21:49:53.282 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:565:24: format: rows: row 21 column 15: 'C' has no terrain, furniture, or other definition

        "##;b_bb_b||||||______;##",
        "##;b_bb_bb|l||||||+||;##",
        "##;;______+l|BC
                       ^
                        SCS_%|/##",
        "###;bbbbbb|l|_______;/##",
        "###;;;;;;;;;;T|T|T|T;###",

21:49:53.282 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:565:26: format: rows: row 21 column 17: 'C' has no terrain, furniture, or other definition

        "##;b_bb_b||||||______;##",
        "##;b_bb_bb|l||||||+||;##",
        "##;;______+l|BCSC
                         ^
                          S_%|/##",
        "###;bbbbbb|l|_______;/##",
        "###;;;;;;;;;;T|T|T|T;###",

21:49:53.283 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:565:31: format: rows: row 21 column 22: '/' has no terrain, furniture, or other definition

        "##;b_bb_b||||||______;##",
        "##;b_bb_bb|l||||||+||;##",
        "##;;______+l|BCSCS_%|/
                              ^
                               ##",
        "###;bbbbbb|l|_______;/##",
        "###;;;;;;;;;;T|T|T|T;###",

21:49:53.283 ERROR : src/mapgen.cpp:4016 [bool mapgen_function_json_base::setup_common(const JsonObject&)] (json-error)
Json error: data/json/mapgen/shelter.json:566:31: format: rows: row 22 column 22: '/' has no terrain, furniture, or other definition

        "##;b_bb_bb|l||||||+||;##",
        "##;;______+l|BCSCS_%|/##",
        "###;bbbbbb|l|_______;/
                              ^
                               ##",
        "###;;;;;;;;;;T|T|T|T;###",
        "############;;;;;;;;;###"
```

**Comment by haveric** [comment]
> This is the error message in the failed MODs loading test:

Shoot, I was hoping to avoid updating the No Hope one, but I'll take a look later today hopefully and see if I can get it updated as well.

**Comment by haveric** [comment]
Updated the shelters for No Hope and Dark Skies Above. I did my best to keep them spawning as intended, but am open to modifications if necessary.
